api/grpc/types/api.proto

```diff
@@ -3,137 +3,138 @@ syntax = "proto3";
 package types;
 
 service API {
-	rpc CreateContainer(CreateContainerRequest) returns (CreateContainerResponse) {}
-	rpc UpdateContainer(UpdateContainerRequest) returns (UpdateContainerResponse) {}
-	rpc Signal(SignalRequest) returns (SignalResponse) {}
-	rpc AddProcess(AddProcessRequest) returns (AddProcessResponse) {}
-	rpc CreateCheckpoint(CreateCheckpointRequest) returns (CreateCheckpointResponse) {}
-	rpc DeleteCheckpoint(DeleteCheckpointRequest) returns (DeleteCheckpointResponse) {}
-	rpc ListCheckpoint(ListCheckpointRequest) returns (ListCheckpointResponse) {}
-	rpc State(StateRequest) returns (StateResponse) {}
-	rpc Events(EventsRequest) returns (stream Event) {}
-	rpc GetStats(StatsRequest) returns (stream Stats) {}
+  rpc CreateContainer(CreateContainerRequest) returns (CreateContainerResponse) {}
+  rpc UpdateContainer(UpdateContainerRequest) returns (UpdateContainerResponse) {}
+  rpc Signal(SignalRequest) returns (SignalResponse) {}
+  rpc AddProcess(AddProcessRequest) returns (AddProcessResponse) {}
+  rpc CreateCheckpoint(CreateCheckpointRequest) returns (CreateCheckpointResponse) {}
+  rpc DeleteCheckpoint(DeleteCheckpointRequest) returns (DeleteCheckpointResponse) {}
+  rpc ListCheckpoint(ListCheckpointRequest) returns (ListCheckpointResponse) {}
+  rpc State(StateRequest) returns (StateResponse) {}
+  rpc Events(EventsRequest) returns (stream Event) {}
+  rpc GetStats(StatsRequest) returns (stream Stats) {}
+}
+
+message IO {
+  string stdin = 1; // path to the file where stdin will be read (optional)
+  string stdout = 2; // path to file where stdout will be written (optional)
+  string stderr = 3; // path to file where stderr will be written (optional)
+  string console = 4; // path to the console for a container (optional)
 }
 
 message CreateContainerRequest {
-	string id = 1; // ID of container
-	string bundlePath = 2; // path to OCI bundle
-	string stdin = 3; // path to the file where stdin will be read (optional)
-	string stdout = 4; // path to file where stdout will be written (optional)
-	string stderr = 5; // path to file where stderr will be written (optional)
-    string console = 6; // path to the console for a container (optional)
-	string checkpoint = 7; // checkpoint name if you want to create immediate checkpoint (optional)
+  string id = 1; // ID of container
```

Comment:
I just lazily do vim gg=G but ya it messes up sometimes

On Tuesday, January 5, 2016, Alexander Morozov <notifications@github.com>
wrote:

> In api/grpc/types/api.proto
> <https://github.com/docker/containerd/pull/48#discussion_r48894868>:
>
> >  }
> >
> >  message CreateContainerRequest {
> > -	string id = 1; // ID of container
> > -	string bundlePath = 2; // path to OCI bundle
> > -	string stdin = 3; // path to the file where stdin will be read (optional)
> > -	string stdout = 4; // path to file where stdout will be written (optional)
> > -	string stderr = 5; // path to file where stderr will be written (optional)
> > -    string console = 6; // path to the console for a container (optional)
> > -	string checkpoint = 7; // checkpoint name if you want to create immediate checkpoint (optional)
> > +  string id = 1; // ID of container
>
> Yeah, it's superhard without gofmt :/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/containerd/pull/48/files#r48894868>.
>